src/coreclr/jit/unwind.cpp

```diff
@@ -165,7 +165,12 @@ void Compiler::unwindPushPopCFI(regNumber reg)
 #endif
         ;
 
+#if defined(TARGET_ARM)
+    createCfiCode(func, cbProlog, CFI_ADJUST_CFA_OFFSET, DWARF_REG_ILLEGAL,
+                  reg >= REG_FP_FIRST ? 2 * REGSIZE_BYTES : REGSIZE_BYTES);
+#else
     createCfiCode(func, cbProlog, CFI_ADJUST_CFA_OFFSET, DWARF_REG_ILLEGAL, REGSIZE_BYTES);
```

Comment:
Are we assuming that we are never going to see SIMD or floating-point registers here for other platforms? Assert may be nice.

Reply:
That assumption is currently true. This is only called on ARM and in one branch of x64 with integer registers. I agree that assert would be nice...